tests/unit/tree/endpoint_node/test_repr.py

```diff
@@ -0,0 +1,21 @@
+from pytest import mark
+
+from scanapi.tree import EndpointNode
+
+
+@mark.describe("endpoint node")
+@mark.describe("__repr__")
+class TestName:
```

Comment:
I believe here we don't need to test the `name` function since it is already being tested in the test_name.py file:

https://github.com/scanapi/scanapi/blob/main/tests/unit/tree/endpoint_node/test_name.py

What we could do is to mock the `name` function, returning a mocked value and check the final result of `__repr__`.

What do you think?

Reply:
Thank you very much for your help! I provided a commit request. I formatted test_repr.py and renamed the test class to "ClassRepr". When it comes to mocking i see, that we could to it, because in the repr function we use the name function of EndpointNode Class and so there is a dependency. However I didn't do it, because I looked up other tests and in such cases there was no mocking used. That's why I didn't mock name function in this unit test.